### PR TITLE
SAAS-1161 migrate the login page to MUI

### PR DESCRIPTION
**Pull request by nicolalamacchia**

### CODE REVIEW

File: src/pages/Login/Login.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React, { FC, useCallback } from 'react';
-import { useStyles } from '@grafana/ui';
 import { OktaSignInWidget } from 'components/OktaSignInWidget';
 import { authConfig } from 'core';
+import { useStyles } from 'core/utils';
 import { useOktaAuth } from '@okta/okta-react';
 import logoImg from 'assets/percona-platform-logo.svg';
-import { logger } from '@percona/platform-core';
+import * as logger from 'core/logger';
```
Comment (fabio-silva) [inline src/pages/Login/Login.tsx:7]: This is something we can probably avoid.
Check https://github.com/percona/grafana-dashboards/blob/main/pmm-app/src/shared/core/logger.ts

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: src/pages/Login/Login.styles.ts [hunk 2]
```diff
@@ -1,183 +1,178 @@
...
+      text-decoration: none;
+      align-items: center;
+      line-height: 30px;
+      vertical-align: middle;
       display: flex;
 
       img {
-        width: 100%;
-        max-width: 350px;
+        margin-left: 7px;
```
Comment (TomasBlazek3PG) [inline src/pages/Login/Login.styles.ts:47]: please use spacing

File: src/pages/Login/Login.styles.ts [hunk 3]
```diff
@@ -1,183 +1,178 @@
...
+        margin-right: ${spacing(3)};
       }
-    `,
-    texts: css`
-      color: ${palette.white};
+    }
+
+    h2 {
+      font-size: 32px;
```
Comment (TomasBlazek3PG) [inline src/pages/Login/Login.styles.ts:53]: please use predefined sizes

File: src/pages/Login/Login.tsx [hunk 4]
```diff
@@ -28,12 +28,12 @@ export const LoginPage: FC = () => {
...
-      <div className={styles.leftSide}>
-        <div className={styles.logo}>
+    <main data-testid="login-page-container" className={styles.container}>
+      <div data-testid="login-page-left-side-wrapper" className={styles.leftSide}>
+        <div data-testid="login-page-logo" className={styles.logo}>
           <img src={logoImg} alt={Messages.logoAlt} />
         </div>
-        <div className={styles.texts}>
+        <div data-testid="login-page-title" className={styles.texts}>
           <h2>{Messages.title}</h2>
```
Comment (TomasBlazek3PG) [inline src/pages/Login/Login.tsx:37]: Would use Typography component

**Review by TomasBlazek3PG — CHANGES_REQUESTED** [review]
Using of the predefined sizes is all over the styles so I didnt mention it in every place.

File: src/pages/Login/Login.styles.ts (hunk 2 above)
Comment (nicolalamacchia) [reply src/pages/Login/Login.styles.ts:47]: this is a nit and that's from the existing code, out of the scope of this PR. Also this is likely going to go away after the redesign.

File: src/pages/Login/Login.styles.ts (hunk 3 above)
Comment (nicolalamacchia) [reply src/pages/Login/Login.styles.ts:53]: this is a nit and that's from the existing code, out of the scope of this PR. Also this is likely going to go away after the redesign.

File: src/pages/Login/Login.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply src/pages/Login/Login.tsx]: good point, thanks! done

File: src/pages/Login/Login.tsx (hunk 4 above)
Comment (nicolalamacchia) [reply src/pages/Login/Login.tsx]: good point, done

**Pull request merged**